src/DenseACSets.jl

```diff
@@ -644,6 +644,26 @@ end
 ACSetInterface.cascading_rem_parts!(acs::ACSet, type, parts) =
   delete_subobj!(acs, Dict(type=>parts))
 
+function ACSetInterface.undefined_subparts(acs::StructACSet{S,Ts}, f::Symbol) where {S,Ts}
+  s = Schema(S)
+  f ∈ homs(s; just_names=true) || error("$f is not a hom")
+  _undefined_subparts(acs, f, acs.parts[dom(s,f)])
+end
+
+function ACSetInterface.undefined_subparts(acs::DynamicACSet, f::Symbol)
```

Comment:
It might be unnecessary to make a different method for DynamicACSets. You can get the schema for any ACSet by calling `acset_schema(acs)`.